packages/eslint-plugin-template/src/rules/role-has-required-aria-props.ts

```diff
@@ -0,0 +1,95 @@
+import type {
+  TmplAstTextAttribute,
+  TmplAstElement,
+} from '@angular-eslint/bundled-angular-compiler';
+import type { ARIARoleDefintionKey } from 'aria-query';
+import { roles } from 'aria-query';
+import {
+  createESLintRule,
+  getTemplateParserServices,
+} from '../utils/create-eslint-rule';
+import { getDomElements } from '../utils/get-dom-elements';
+import { toPattern } from '../utils/to-pattern';
+import { isSemanticRoleElement } from '../utils/is-semantic-role-element';
+
+type Options = [];
+export type MessageIds = 'roleHasRequiredAria' | 'suggestRemoveRole';
+export const RULE_NAME = 'role-has-required-aria-props';
```

Comment:
I ported this rule from [eslint-plugin-jsx-a11y role-has-required-aria-props](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/role-has-required-aria-props.md) and used the same name, but I've considered dropping "-props" from the name and just calling it "role-has-required-aria" since props is a React convention. Let me know if that are opinions one way or another.

Reply:
renamed to `accessibility-role-has-required-aria`